components/common/Text.tsx

```diff
@@ -0,0 +1,46 @@
+import { useFonts } from 'expo-font';
+import { useContext } from 'react';
+import { StyleProp, StyleSheet, Text, TextStyle } from 'react-native';
+import { FontLoadedContext } from '../../App';
+import TitleSizes from '../../theme/fontSizes';
+
+interface TextProps {
+  children: React.ReactNode;
+  size?: number;
+  style: StyleProp<TextStyle>;
+  centered?: boolean;
+}
+
+const LText = ({ children, size = 12, style, centered }: TextProps) => {
+  const textAlign = centered ? 'center' : 'left';
+  const fontsLoaded = useContext(FontLoadedContext);
```

Comment:
A minor comment, could you try to keep the hook statements on top of the component in general. For example, we can carry this `useContext` to the top.